packages/cosmic-swingset/test/test-make.js

```diff
@@ -43,3 +68,92 @@ test('make and exec', async t => {
     }),
   );
 });
+
+const query = async args => {
+  const parts = [];
+  const run = new Promise((resolve, reject) => {
+    const cmd = spawn(
+      'bin/ag-chain-cosmos',
+      ['--chain-id=agoric', 'query', ...args],
+      { cwd: `${dirname}/..` },
+    );
+    cmd.stdout.on('data', chunk => parts.push(chunk));
+    cmd.on('exit', code => {
+      if (code !== 0) {
+        reject(Error(`exit ${code} from: query ${args}`));
+        return;
+      }
+      const txt = parts.join('');
+      if (txt.trim() === '') {
+        reject(Error(`empty output from: query ${args}`));
+        return;
+      }
+      const info = JSON.parse(txt);
+      resolve(info);
+    });
+  });
+  return run;
+};
+
+const waitForBlock = async height => {
+  const delay = ms => new Promise(resolve => setTimeout(resolve, ms));
+
+  for (;;) {
+    try {
+      const info = await query(['block']);
+      if (Number(info.block.header.height) >= height) {
+        return;
+      }
+    } catch (reason) {
+      console.warn(reason);
+      await delay(5_000);
+    }
+  }
+};
+
+test.serial('PSM bootstrap and provision', async t => {
+  // XXX coordinate with vbank/provision from golang
+  const pool = 'agoric1megzytg65cyrgzs6fvzxgrcqvwwl7ugpt62346';
```

Comment:
where does this string come from? I see the comment about coordinating, which made me think it was from that part of the codebase but I don't see it anywhere.